examples/routing_service/udp_socket_adapter_typed/README.md

```diff
@@ -5,6 +5,8 @@
 This example shows how to implement a simple Routing Service Adapter plugin
 in C++11 to receive data from a UDP socket using RTI Routing Service.
 
+This example requires including a Types.xml file with the data type information.
```

Comment:
While we're using Dynamic Data for the outside endpoints, is it strictly necessary to provide an XML types file? What about when we use `route_types = true` in Routing Service? I believe it's the default now. Have you tested without XML types?